pom.xml

```diff
@@ -214,7 +214,7 @@
     <commons-crypto.version>1.1.0</commons-crypto.version>
     <commons-cli.version>1.5.0</commons-cli.version>
     <bouncycastle.version>1.60</bouncycastle.version>
-    <tink.version>1.7.0</tink.version>
+    <tink.version>1.8.0</tink.version>
```

Comment:
@bjornjorgensen We can exclude `androidx.annotation:annotation` from `tink`, and then we should be able to remove the newly added maven repository. I have tested it and it should be that only `androidx.annotation:annotation` cannot be downloaded from the central repository and exclude does not cause UT failures.

Reply:
yes, there may be a way to do it. I see that maybe they have a fix for this https://github.com/tink-crypto/tink-java/commit/5733ab346c95eac6840bff23e050a1e16749e928 . Considering the KISS principle, I'm waiting for a new release. It can get a bit complicated if we need to troubleshoot.

Reply:
Got it, we can reuse this jira when upgrading the new release